src/net/sourceforge/kolmafia/Modifiers.java

```diff
@@ -2629,10 +2509,6 @@ private static <T extends net.sourceforge.kolmafia.modifiers.Modifier> String pa
 
         String value = matcher.group(1);
 
-        if (tag.equals("Class")) {
```

Comment:
This is no longer needed because there is no longer a "table" of StringModifiers any more, right? Just Booleans? So this particular method no longer applies to StringsModifiers.

Reply:
yes: Class is a string modifier, so there's no point doing the check when you know you've a modifier of different type